templates/definition/charger/victron-ev-chaging-station.yaml

```diff
@@ -0,0 +1,52 @@
+template: victron-ev-charging-station
+products:
+  - brand: Victron
+    description:          
```

Comment:
Die trailing Spaces müssen weg. Oder einfach prettier benutzen (sollte so auch in der README stehen)

Reply:
Das scheint nicht zu reichen.. Habe prettier benutzt, aber es schlägt immer noch fehl.

Reply:
Schau mal in die Meldungen

Reply:
Scheint jetzt besser zu sein. Nur clean mecket noch...

Reply:
Der meckert immer noch über das fehlende newline, obwohl das jetzt drin ist. Lint selber läuft ja auch.